pkg/operator/ceph/controller/cluster_info.go

```diff
@@ -170,6 +190,21 @@ func CreateOrLoadClusterInfo(clusterdContext *clusterd.Context, context context.
 	return clusterInfo, maxMonID, monMapping, nil
 }
 
+func genAdminKeyrings(context *clusterd.Context, namespace string) (*cephclient.CephCred, error) {
+	dir := path.Join(context.ConfigDir, namespace)
+	if err := os.MkdirAll(dir, 0744); err != nil {
+		return nil, errors.Wrapf(err, "failed to create dir %s", dir)
+	}
+
+	operatorSecret, err := genSecret(context.Executor, dir, cephclient.AdminUsername, adminCapArgs)
```

Comment:
`genSecret()` calls the `ceph-authtool`, which generates a key but doesn't save it to the ceph auth list. Try calling [cephclient.AuthGetOrCreateKey()](https://github.com/rook/rook/blob/master/pkg/operator/ceph/client/controller.go#L241) to create a key. This is the same API called by the CephClient controller for creating cephx keys that show up in `ceph auth ls`.